tests/torch/pruning/helpers.py

```diff
@@ -669,6 +670,27 @@ def forward(self, x):
         return x
 
 
+class ResUnetblock(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.conv0 = nn.Conv3d(8, 32, 1)
+        self.in0 = nn.InstanceNorm3d(32, affine=True)
+        self.conv1 = nn.Conv3d(32, 32, 1)
+        self.in1 = nn.InstanceNorm3d(32, affine=True)
+        self.conv2 = nn.Conv3d(32, 32, 1)
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        x = self.conv0(x)
+        x1 = self.in0(x)
+        x1 = F.leaky_relu(x1)
+        x1 = self.conv1(x1)
+        x1 = self.in1(x1)
+        x1 = F.leaky_relu(x1)
+        x1 = self.conv2(x1)
+
+        return x + x1
```

Comment:
Please consider to add conv layer to the end of model to enable first pruning group to cover all cases of the unet block pruning

Reply:
Done